cli/lib/kontena_cli.rb

```diff
@@ -1,3 +1,12 @@
+begin
+  unless Gem.loaded_specs.include?('kontena-cli')
+    load File.expand_path('../../kontena-cli.gemspec', __FILE__)
+    KontenaCliGemspec.activate
+    KontenaCliGemspec.activate_dependencies
+  end
+rescue LoadError
```

Comment:
Gems don't always include their own .gemspec file. We do by chance (because the file list is built using `git ls files` (which by the way adds a bunch of unnecessary files such as the omnibus dir, Dockerfile, Gemfile, etc and makes it impossible to `gem build` the cli unless you downloaded it using git))

Gem::Specification has `#to_ruby` which builds a .gemspec from scratch.

Gemspecs are installed to `$GEM_HOME/specifications/` generated using that `#to_ruby`, so it won't include any of the magic from the .gemspec in cli root.

Since loading it is not absolutely essential, escpecially not for a regular joe, it's better to just ignore the load error in case it happens.